src/renderers/shared/fiber/ReactFiberDevToolsHook.js

```diff
@@ -12,15 +12,30 @@
 
 'use strict';
 
+var emptyFunction = require('fbjs/lib/emptyFunction');
 var warning = require('fbjs/lib/warning');
 
 import type {Fiber} from 'ReactFiber';
 import type {FiberRoot} from 'ReactFiberRoot';
 
 declare var __REACT_DEVTOOLS_GLOBAL_HOOK__: Object | void;
 
+let didCatchErrors = false;
+function tryCall(fn, rendererID, arg) {
+  try {
+    fn(rendererID, arg);
+  } catch (err) {
+    // Catch all errors from DevTools because throwing might be unsafe
+    if (__DEV__) {
+      warning(!didCatchErrors, 'React DevTools encountered an error: %s', err);
```

Comment:
I don’t think so..

If the first argument is `false`, warning is printed.
I want to print the warning if we have *not* printed it before.

Therefore I want the argument to be `false` when we have *not* printed it, or `true` when we have printed it before. Therefore I pass `didCatchErrors`.